### PR TITLE
Supermatter chad during April Fools

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
An image posted by a friend of mine compared the Supermatter Shard to a banana on a pillar. In my madness I have noticed it was more akin to the typical Chad meme that will be dead if you are reading this 10 years in the future, at least if it didn't get the coolface (trollface) treatment of having its grave desecrated and pissed on by millions of children who have run it through the "epic ironic humor" treatment where it became a dumb creepypasta material. In Deus Ex: Mankind Divided if you decide to do the mission where you go after the Orchid in the VersaLife vault instead of going after the terrorist and daughter of a local person, one of the canisters, although initially unlit, actually contains a clone of Adam Jensen, implying that the Adam you play as during the game is actually a clone, due to their lack of memories and the fact that their augments are completely different even if they look similar, sent to act as a sleeper agent until you get close enough to Janus to kill him. However, the game itself was pissed on due to interference from businessmen so probably nothing will actually happen with the Deus Ex franchise for at least a few more years after the creation of this pull request, and additionally the Breach gamemode sucks and I don't know how the heck anyone thought it should be an entire alternate gamemode when people wanted the main story and gameplay far more than they wanted this.  Actual manpower spent on making Breach could've been used to optimize the game so that it doesn't take literally a minute every zone transition to load the game.

:cl:
 * tweak: Updated localization files.

### CODE REVIEW

File: code/modules/supermatter/supermatter.dm [hunk 1]
```diff
@@ -102,6 +102,13 @@
 	. = ..()
 	radio = new (src)
 
+/obj/machinery/power/supermatter/shard/New()
+	. = ..()
+	if(Holiday == APRIL_FOOLS_DAY)
+		icon_state = "darkmatter_shard_chad"
+		base_icon_state = "darkmatter_shard_chad"
+		desc = "A strangely translucent and iridescent crystal that looks like it used to be part of a larger structure. <span class='warning'>You are confident this is literally the best engine on the station, no other engine can compare to the intelligence required to set it up nor the unparalleled power output. All those idiot engineers will set up the Singularity, the TGE, the AME, but they all kneel to those who set up the SME. What are you waiting for? If this doesn't produce enough power to power the station for billions of years then you are doing it wrong.</span>"
```
Comment (Exxion) [inline code/modules/supermatter/supermatter.dm]: >TGE

Comment (boy2mantwicethefam) [reply code/modules/supermatter/supermatter.dm]: Terminally generic electricity

Comment (boy2mantwicethefam) [reply code/modules/supermatter/supermatter.dm]: For real, will fix.

**Comment by Eneocho** [comment]
Your insanity has gone too far, B2M. You are dealing with matters no spessman should.

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
there's a rogue pixel on the new supermatter sprites

**Comment by boy2mantwicethefam** [comment]
If you mean the blue one it's intentional @MadmanMartian

**Comment by MadmanMartian** [comment]
On the animated glow one. Appears for a few frames

**Comment by ghost** [comment]
change the user attack texts end thing to say "Your final thoughts are "Ouch!"" and its sold

**Comment by Kurfursten** [comment]
White pixel top left corner in animation only in a few frames, as he said

**Comment by boy2mantwicethefam** [comment]
I looked at the sprites and there's nothing wrong, the white pixel only appears in the bot's image when zoomed in, but the animation itself is clean, meaning the bot's image has image noise.

**Pull request merged**